scripts/doctor

```diff
@@ -42,7 +42,16 @@ else
     count_in_path "bundler"
     if [ $found -eq 0 ]; then
         exitcode=1
-        warn "yel" "   bundler not found. Run 'gem install bundler' to install it it."
+        warn "yel" "   bundler not found. Run 'gem install bundler' to install it."
+    else
+        warn "grn" "   found!"
+    fi
```

Comment:
Why check bundler at all? ... I'm not sure if this is right fix.. Would could just change our scripts to use bundler instead. bundle and bundler are just shims from what I can see.

I'm guessing you had some sort of conflicting application in path with the name bundle. If thats common, switching to bundler should resolve it. I think it's maybe better to do so since bundler will be the less common phrase.